frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx

```diff
@@ -273,7 +320,14 @@ function OutputForDocModal({
             columns={columns}
             dataSource={rows}
             pagination={{ pageSize: 5 }}
-            loading={isLoading}
+            loading={{
+              indicator: (
+                <div>
```

Comment:
@jagadeeswaran-zipstack I don't think we need this additional `div`. Please check and remove this if not required.

Reply:
Yes, We need that div there if not spinner not rendering.